core/src/main/java/org/elasticsearch/action/admin/indices/create/CreateIndexResponse.java

```diff
@@ -32,28 +33,37 @@
 public class CreateIndexResponse extends AcknowledgedResponse {
 
     private boolean shardsAcked;
+    private String index;
+    private Version minimumVersionWithIndex = Version.V_5_6_0;
```

Comment:
@jasontedor, hmmm, that way they will not diverge. Also variable name explains what's going on. But as you say :)